src/engine/Settings.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Godot;
+using Godot;
```

Comment:
Did you add a BOM here? The files should be without that.

Reply:
I don't know about that but when I remove using Godot;  it gave me an error

Reply:
BOM is an invisible character that a file can begin with.
https://en.wikipedia.org/wiki/Byte_order_mark

When a diff shows no changed visible characters, and it is the first line, my first guess is that your editor added a BOM, our source files don't use that so you need to edit the file to remove it.

Reply:
Opened an issue to make an automatic check for this: https://github.com/Revolutionary-Games/Thrive/issues/1786